.changeset/old-zebras-teach.md

```diff
@@ -0,0 +1,15 @@
+---
+'astro': major
+---
+
+Updates `astro sync` behavior
+
+`astro sync` will no longer create nor update `src/env.dts`. You can keep using it but we recommend you update your root `tsconfig.json` instead:
```

Comment:
```suggestion
Astro's TypeScript configuration has been updated to [[read `include` and `exclude` properties from]](however you're now "using" this config file) `tsconfig.json` by default. This means that `src/env.d.ts` is only necessary if you have added custom ... (whatever we said last time?).

Additionally, running `astro sync` no longer creates, nor updates, `src/env.d.ts` as it is not required for type-checking standard Astro projects.

To update your project to Astro's recommended TypeScript settings, please add the following `include` and `exclude` properties to `tsconfig.json`:
```
just some blah blah up here  with very not specific wording!! Can you work with this?